.gitlab-ci.yml

```
@@ -180,6 +180,12 @@ pkg:docker:
         PLATFORM: linux/amd64
         ARCH: x86_64
         TRIPLET: x86_64-linux-gnu
+        TAG: docker
+      - DISTRO: debian
+        PLATFORM: linux/arm64/v8
+        ARCH: arm64
+        TRIPLET: aarch64-linux-gnu
+        TAG: PI5
```

Can we please use a lower-case slug as a tag?
```suggestion
        TAG: pi5
```